CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+4.60.1
+==========
+### Bug Fixes
+We addressed an issue where attempts to acquire a token via certificate authentication resulted in a `Microsoft.Identity.Client.MsalServiceException (Error code: AADSTS5002730), signaling an "Invalid JWT token. Unsupported key for the signing algorithm."` This was due to a known bug in Microsoft Entra ID (Azure AD) that affects the handling of JWT tokens signed with certain algorithms, specifically SHA2 and PSS. See [4690](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/4690)
```

Comment:
```suggestion
Resolved an issue where MSAL attempts to acquire a token via certificate authentication using SHA2 and PSS resulting in a `MsalServiceException' (Error code: AADSTS5002730). See [4690](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/4690)
```

I think we can make this a bit less technical and detailed. Customers can look at the issue for more info.